hydra-node/src/Hydra/Ledger/Simple.hs

```diff
@@ -111,7 +111,7 @@ instance FromCBOR SimpleTxIn where
 simpleLedger :: Ledger SimpleTx
 simpleLedger =
   Ledger
-    { applyTransactions =
+    { applyTransactions = \slot ->
```

Comment:
This is currently unused and makes CI fail. It's fine to mark this as `_slot` and document that `SimpleTx` transactions don't have a notion of time.

I realize that the module documentation above is severly outdated. Maybe we can use the chance to update it to say what `SimpleTx` is and what it is not (i.e. no time validity)

Reply:
I updated the documentation but I did not find it outdated.

Can you plz give a look to my changes and describe which updates would you introduce to the module description?

Reply:
It refers to a `Mock` transaction which does not exist anymore. I would propose to write something like:


```